pkg/generate/app/cmd/newapp_test.go

```diff
@@ -53,6 +53,17 @@ func TestValidate(t *testing.T) {
 			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
 			parms:               map[string]string{},
 		},
+		"envfile": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: "/tmp/envfile1",
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
+			parms:               map[string]string{},
+		},
```

Comment:
@bparees , @fabianofranz : Need feedback on this. The documentation on test-cases isn't that much clear. I checked the newapp_test.go as suggested and added this block. How do we ensure that this is being run.